cmd/present/local.go

```diff
@@ -85,8 +85,8 @@ func main() {
 		log.Print(localhostWarning)
 	}
 
-	if present.PresenterEnabled {
-		log.Println("Presenter enabled, press 'P' to open Presenter")
+	if present.NotesEnabled {
+		log.Println("Notes enabled, press 'N' from browser to open notes")
```

Comment:
Wording suggestion, perhaps "press 'N' in browser" is a better way to phrase this?